src/main/java/com/mongodb/jdbc/MongoDriver.java

```diff
@@ -139,9 +140,11 @@ private MongoConnection createConnection(ConnectionString cs, Properties info) t
             case MONGOSQL_DIALECT:
                 if (info.containsKey(CONVERSION_MODE)) {
                     throw new SQLClientInfoException(
-                            String.format("must not set '%s' if '%s' is '%s'",
```

Comment:
These are being done by spotless. Not sure how we are passing the spotless test without running spotless